Extension/lib/webrequest.js

```diff
@@ -147,6 +147,13 @@
             return;
         }
 
+        // And we don't need this check on newer than 58 chromes anymore
+        // https://github.com/AdguardTeam/AdguardBrowserExtension/issues/572
+        if (adguard.utils.browser.isChromeBrowser() &&

```

Comment:
Also, we shouldn't override `WebSocket` in the content script, but in this case, you have to check chromium version)